src/NzbDrone.Core/Download/Clients/qBittorrent/QBittorrentSettings.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using FluentValidation;
+using NzbDrone.Core.Annotations;
+using NzbDrone.Core.ThingiProvider;
+using NzbDrone.Core.Validation;
+
+namespace NzbDrone.Core.Download.Clients.QBittorrent
+{
+    public class QBittorrentSettingsValidator : AbstractValidator<QBittorrentSettings>
+    {
+        public QBittorrentSettingsValidator()
+        {
+            RuleFor(c => c.Host).ValidHost();
+            RuleFor(c => c.Port).InclusiveBetween(0, 65535);
+        }
+    }
+
+    public class QBittorrentSettings : IProviderConfig
```

Comment:
@markus101 I pushed a patch that adds a UseSsl setting as the other clients do. I was able to generate a certificate/key, add them to qBittorrent's webui, and test the client to verify that it works.